csrankings-m.csv

```diff
@@ -1582,7 +1585,6 @@ Mi Zhang 0001,Fudan University,https://mi-zhang-fdu.github.io/index.chn.html,lXL
 Mi Zhang 0002,Ohio State University,https://engineering.osu.edu/people/mizhang.1,r3A90uAAAAAJ
 Mia Minnes,Univ. of California - San Diego,https://cseweb.ucsd.edu/~minnes,jr9OS68AAAAJ
 Mian M. Awais,LUMS,https://lums.edu.pk/lums_employee/516,wkxlfFAAAAAJ
-Mian Muhammad Awais,LUMS,https://lums.edu.pk/lums_employee/516,wkxlfFAAAAAJ

```

Comment:
Please explain removal; note that an alias entry exists for the same person immediately above.

Reply:
(Both are needed)

Reply:
I have added both now